velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -155,5 +155,24 @@ TEST_F(ProbabilityTest, binomialCDF) {
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
 
+TEST_F(ProbabilityTest, poissonCDF) {
+  const auto poissonCDF = [&](std::optional<double> lambda,
+                              std::optional<int64_t> value) {
+    return evaluateOnce<double>("poisson_cdf(c0, c1)", lambda, value);
+  };
+
+  EXPECT_EQ(0.91608205796869657, poissonCDF(3, 5));
```

Comment:
Please add tests with std::nullopt values also.

Reply:
Added it